include/libswiftnav/signal.h

```diff
@@ -15,14 +15,9 @@
 
 #include "common.h"
 
-#define GPS_L1_SATS 32
-#define WAAS_SATS 3
-#define EGNOS_SATS 3
-#define GAGAN_SATS 2
-#define MSAS_SATS 2
-#define SDCM_SATS 2
-#define SBAS_SATS (WAAS_SATS + EGNOS_SATS + GAGAN_SATS + MSAS_SATS + SDCM_SATS)
-#define ALL_SATS (GPS_L1_SATS + SBAS_SATS)
+#define NUM_SATS_GPS 32
+#define NUM_SATS_SBAS 22
+#define NUM_SATS (NUM_SATS_GPS + NUM_SATS_SBAS)
 
 #define CONSTELLATION_GPS 0
```

Comment:
Can we make constellation and band enums with CONSTELLATION_COUNT and BAND_COUNT at the bottom?